drawSquareDoubles.c

```diff
@@ -200,26 +192,22 @@ void scaleMulti2DPoints(double arr[8][2]){
             
 void projectPoints2d(double arr[8][3], double p_points[8][2], const double DISTANCE){
     for(int i = 0; i < 8; i++){
-        //extract x and y from array.
+        //extract x, y and z from array.
         double x = arr[i][0];
         double y = arr[i][1];
         double z = arr[i][2];
 
         double zPerspective = 1/(DISTANCE - z);
 
-        // zPerspective = 1 / zPerspective;
-
-        double p_Mat[2][3] = {{1,0,0},{0,1,0}}; //values of 1 give orthgraphic view
-
         double p_Mat2[2][3] = {{zPerspective,0,0},{0,zPerspective,0}};
         
         double x_p2 = (p_Mat2[0][0]*x)+(p_Mat2[0][1]*y)+(p_Mat2[0][2]*z);
         double y_p2 = (p_Mat2[1][0]*x)+(p_Mat2[1][1]*y)+(p_Mat2[1][2]*z);
 
-        double x_p = (p_Mat[0][0]*x)+(p_Mat[0][1]*y)+(p_Mat[0][2]*z);
-        double y_p = (p_Mat[1][0]*x)+(p_Mat[1][1]*y)+(p_Mat[1][2]*z);
+        //double p_Mat[2][3] = {{1,0,0},{0,1,0}}; //values of 1 give orthgraphic view
+        // double x_p = (p_Mat[0][0]*x)+(p_Mat[0][1]*y)+(p_Mat[0][2]*z);
+        // double y_p = (p_Mat[1][0]*x)+(p_Mat[1][1]*y)+(p_Mat[1][2]*z);
```

Comment:
generally don't commit commented code.
not obvious if it's meant to be here or not